container/container_windows.go

```diff
@@ -83,3 +83,13 @@ func cleanResourcePath(path string) string {
 	}
 	return filepath.Join(string(os.PathSeparator), path)
 }
+
+// canMountContainerFS determines if the file system for the container
+// can be mounted locally. In the case of Windows, this is not possible
+// for Hyper-V containers during WORKDIR execution for example.
+func (container *Container) canMountContainerFS() bool {
+	if containertypes.Isolation.IsHyperV(container.HostConfig.Isolation) {
```

Comment:
small nit, could be `return !containertypes.Isolation.IsHyperV(container.HostConfig.Isolation)`

No need to update just for this.